frontend/package.json

```diff
@@ -7,7 +7,17 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc -b && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "build-only": "vite build",
+    "type-check": "vue-tsc --noEmit",
+    "lint:eslint": "eslint . --ext .vue,.ts --ignore-path .gitignore",
+    "lint:prettier": "prettier --check --cache '**/*.{vue,js,ts,json}'",
+    "lint": "npm-run-all lint:**",
+    "format": "npm-run-all format:**",
+    "format:eslint": "eslint --fix . --ext .vue,.ts --ignore-path .gitignore",
+    "format:prettier": "prettier --write --cache '**/*.{vue,js,ts,json}'",
+    "test": "playwright test",
+    "test:setup": "playwright install --with-deps"
```

Comment:
I think some of these were deliberately moved to the root's package.json (cc @lubej), so I would keep the prettier stuff there. Even the backend prettier should be kept there.